.github/workflows/iroha2-dev-pr-unstable.yml

```diff
@@ -1,4 +1,4 @@
-name: I2::Dev::Tests (Unstable)
```

Comment:
Actually, let's get rid of this workflow, and move it as a separate `job` inside `I2::Dev::Tests`.

Reply:
The original reason we had this as a separate workflow was because we couldn't restart the failed jobs, just failed workflows. Now that we can re-run failed jobs, we should merge it back.